app/models/git/track.rb

```diff
@@ -36,6 +36,21 @@ def snippet
       repo.read_text_blob(commit, snippet_filepath)
     end
 
+    memoize
+    def debug
+      repo.read_text_blob(commit, debug_filepath)
+    end
+
+    memoize
+    def help
+      repo.read_text_blob(commit, help_filepath)
+    end
+
+    memoize
+    def tests
+      repo.read_text_blob(commit, tests_filepath)
+    end
```

Comment:
You can metaprogram all this. Something like this:

```ruby
%i[about snippet debug help tests].each do |meth|
  define_method meth do
    iv = "@__#{meth}__"
    return instance_variable_get(iv) if instance_variable_defined?(iv)
    instance_variable_set(iv, repo.read_text_blob(commit, meth)
  end
end
```

Or you could go one step funkier and DSL it:
```ruby
def self.text_blob_reader(*meths)
  meths.each do |meth|
    define_method meth do
      iv = "@__#{meth}__"
      return instance_variable_get(iv) if instance_variable_defined?(iv)
      instance_variable_set(iv, repo.read_text_blob(commit, meth)
    end
  end
end

text_blob_reader :about snippet debug help tests
```